src/store/reducers.js

```diff
@@ -105,11 +124,7 @@ function entityDeleted(state, { meta }) {
     };
 }
 
-function onEntitiesLoaded(state, { payload, meta }) {
-    if (meta?.lastDateRequest < state?.lastDateRequest) {
-        return state;
-    }
```

Comment:
@rvsia this was added to prevent race conditions, right? I don't think we can remove this.